### ISSUE TITLE
Option-clicking checkbox doesn't work as expected.

### ISSUE THREAD

**Issue by aanklewicz** [hjuutilainen/munkiadmin#70]
In most OS X apps when you option click on a checkbox in a list, it will check or uncheck all the contents. Tried doing that in catalogs and it didn't work.

**Comment by hjuutilainen** [comment]
True, will add at some point. In the meantime, you can right-click anywhere on the packages list (in Catalogs view) and select enable all.
